pkgs/applications/networking/synology-drive-client/default.nix

```diff
@@ -30,7 +30,7 @@ let
 
     src = fetchurl {
       url = "${baseUrl}/${version}/Ubuntu/Installer/synology-drive-client-${buildNumber}.x86_64.deb";
-      sha256 = "sha256-Zf6JMghXy8ODbR4MhVSPmD4QDu003MTc7YNfbiRVRoY=";
+      sha256 = "sha256-Spl6DC+wf+JaXjwH2ecraySo1VtA+EiI3/TWw9UOSA8=";
```

Comment:
Should sha for mac version also be updated?

Reply:
Not really familiar with darwin, what i just noticed is that the following page looks rather empty:

https://hydra.nixos.org/job/nixos/trunk-combined/nixpkgs.synology-drive-client.x86_64-darwin

In comparison x86_64-linux has some jobs:

https://hydra.nixos.org/job/nixos/trunk-combined/nixpkgs.synology-drive-client.x86_64-linux

So it probably would be good if there was some verification wheter it even works without the bump.

Not sure about the policies regarding this but one could also consider dropping darwin support ...